README.md

```diff
@@ -6,30 +6,25 @@ A badge with two available themes to help users discover new widgets and content
 
 ## Getting Started
 
-Add the following import to your page's `<head>`:
+Include the component in your page:
 
 ```html
-<script type="module" src="https://unpkg.com/@myuw-web-components/myuw-badge@^1?module"></script>
-<script nomodule scr="https://unpkg.com/@myuw-web-components/myuw-badge@^1"></script>
-```
-
-**Important**: For responsiveness to many screen sizes, ensure you have included the viewport meta tag in your application:
```

Comment:
I don't know what this has to do with a little badge icon, and we don't do it in the demo, so I'm assuming it's not important.